### ISSUE TITLE
The `options` for the `mountpoint` are missing in template

### ISSUE THREAD

**Issue by citnfm** [robertdebock/ansible-role-autofs#1]
## Describe the bug
The `options` specified for the `mountpoint` (not the `directories`) configuration are **not** being added to the `/etc/auto.master.d/00-*` file.

Based on the examples given in the REAMDE this should be an available feature:

```yaml
#   - mountpoint: /net
#     options:
#       - "--timeout=60"
```

## Playbook
Requirements file for `ansible-galaxy` import:
```yaml
---
roles:
  - name: robertdebock.autofs
    version: "1.2.6"
```

Playbook:
```yaml
---
- name: Deploy autofs mounts
  hosts: all
  become: yes
  gather_facts: yes

  roles:
    - role: robertdebock.autofs
```

`autofs_maps` variable:
```yaml
---
autofs_maps:
  - mountpoint: /home
    options:
      - "--timeout=60"  # <-- This is not being added
    directories:
      - path: "*"
        server: "nfs.server.example:/&"
        options:
          - rw
          - nfsvers=4
          - exec
          - nosuid
          - sync
          - rsize=8192
          - wsize=8192
```

## Output

I don't believe the playbook output is relevant here, instead here are the files created:
```
root:~ # cat /etc/auto.master.d/00-home.autofs
#
# Ansible managed
#
/home /etc/auto.home
```
```
root:~ # cat /etc/auto.home 
#
# Ansible managed
#
* -rw,nfsvers=4,exec,nosuid,sync,rsize=8192,wsize=819 nfs.server.example:/&
```

## Problem File
The problem appears to be in [`templates/template.autofs.j2` on line 2](https://github.com/robertdebock/ansible-role-autofs/blob/1de434bc522d83f4e40eae88e0e79bf7e28f89d1/templates/template.autofs.j2#L2). The template does not have any logic for including `options` specified for the `mountpoint`. 

A solution would be to update it to loop through the `options` for the `mountpoint` and add them if present (based on how the `templates/map.j2` template works):
```yaml
{{ item.mountpoint }} /etc/auto.{{ item.mountpoint | regex_replace('/', '') }} {% if item.options is defined %}{% for option in item.options %}{{ option }}{% if not loop.last %} {% endif %}{% endfor %} {% endif %}
```

I am making the assumption that the options are separated by a space and not a comma. I can't find any reference material that explicitly states which one it is.

## Environment

- Control node OS: `RHEL 7.9`
- Control node Ansible version: `2.10.4`
- Managed node OS: `RHEL 7.9`

**Comment by robertdebock** [comment]
Good find, your PR has been merged, I'll release a new version and close the issue.

Thanks for your pro-active behaviour!

**Comment by robertdebock** [comment]
Fixed with version 1.2.7.
